modules/store.js

```diff
@@ -0,0 +1,33 @@
+// Store Class: Handles Storage
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  static addBook(book) {
+    const books = Store.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  static removeBook(author) {
+    const books = Store.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.author === author) {
+        books.splice(index, 1);
```

Comment:
- [ ] [OPTIONAL] As I've mentioned previously please favor using array methods that don't mutate the data directly.
- [ ] For this you could use:
  ```
   books = books.filter(({ _author }) => _author !== author);
   ```